app/Http/Controllers/EntryController.php

```diff
@@ -15,7 +16,13 @@ class EntryController extends Controller
      */
     public function index(Request $request)
     {
-        $entries = Entry::all();
+        if ($userId = $request->get('user_id')) {
+            $entries = Entry::where('user_id', $userId)->get();
+        } elseif (is_null($request->get('user_id'))) {
```

Comment:
the request class has a function for this already, iirc; check the docs to be sure but i think its just:

```suggestion
        } elseif ($request->has('user_id')) {
```

Reply:
Actually given my other comment about the else, you can probably just make this the `else` instead

Reply:
Done.